lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -7103,6 +7114,23 @@ namespace Js
 #endif
     }
 
+    bool FunctionBody::DoInterpreterProfileWithLock() const
+    {
```

Comment:
only for background thread. do you think there's other instance that background jit will call into DoInterpreterProfile ?